FlowCrypt/src/main/java/com/flowcrypt/email/api/retrofit/RetrofitApiServiceInterface.kt

```diff
@@ -113,32 +114,25 @@ interface ApiService {
   suspend fun checkPolicyForWkdDirect(@Path("directHost") directHost: String): Response<ResponseBody>
 
   /**
-   * This method calls "https://flowcrypt.com/api/account/get"
+   * Get client configuration
    */
-  @POST(BuildConfig.API_URL + "account/get")
-  suspend fun getClientConfigurationFromFlowCryptComBackend(
+  @GET("https://{baseFesUrlPath}/api/v1/client-configuration?domain={domain}")
+  suspend fun getClientConfigurationFromFes(
```

Comment:
The EKM/Attester/FES etc groups on  ApiClientRepository looked good.

Since here we cannot do that, can you rename all of the methods in this file to start with the service name?

getClientConfigurationFromFes -> fesGetClientConfiguration
submitPrimaryEmailPubKey -> attesterSubmitPrimaryEmailPubkey

I know there is some value in having the methods with exactly the same name, but at the same time the clarity of what is being called is also important.